1.2/spec.md

```diff
@@ -627,10 +627,34 @@ Issue: [ocr_carea vs ocrx_block](https://github.com/kba/hocr-spec/issues/28)
 
 ### `ocrx_line`
 
-Issue: [ocr_line vs ocrx_line](https://github.com/kba/hocr-spec/issues/19)
-
-  * any kind of "line" returned by an OCR system that differs from the standard ocr_line above
+  * any kind of "line" returned by an OCR system that differs from [[#ocr_line]]
   * might be some kind of "logical" line
+  * examples include line continuations and rowspan in tables
+
+<div class="example">
+
+Consider the following snippet, containing a wide-spaced heading broken over
+two physical lines:
+
+<figure>
+  <img
+    width=600
+    alt="Wide spaced two line heading"
+    src="../images/akf-widespaced-heading.png"/>
+</figure>
+
+An OCR engine could produce the following output, indicating the two physical
+lines that form a single logical line:
+
+```html
+...
+<span class="ocrx_line">
```

Comment:
It's `ocr_line` nested in `ocrx_line`, in this case a single heading split over two lines.

But I'll gladly make a better example if you have an idea. What i've seen in the wild is just replacements for `ocr_line`, e.g. https://github.com/jwilk/ocrodjvu/blob/master/lib/hocr.py.